shared/desktop/remote/component-loader.desktop.js

```diff
@@ -1,106 +1,67 @@
 // @flow
 // This loads up a remote component. It makes a pass-through store which accepts its props from the main window through ipc
 // Also protects it with an error boundary
-import React, {Component} from 'react'
+import * as React from 'react'
+import * as SafeElectron from '../../util/safe-electron.desktop'
+import * as Styles from '../../styles'
 import ReactDOM from 'react-dom'
 import RemoteStore from './store.desktop'
 import Root from '../renderer/container.desktop'
-import Menubar from '../../menubar/remote-container.desktop'
```

Comment:
this component used to know about everything and just switched between the remote types, now its more generic and expects stuff to be passed in